alerttags/custom_payloads/index.xml

```diff
@@ -4,7 +4,7 @@
     <title>CUSTOM_PAYLOADS on ZAP</title>
     <link>/alerttags/custom_payloads/</link>
     <description>Recent content in CUSTOM_PAYLOADS on ZAP</description>
-    <generator>Hugo -- gohugo.io</generator>
+    <generator>Hugo 0.125.0</generator>
```

Comment:
We might need to do some changes to hugo configs to not add the version, otherwise we are going to have noise PRs.